securitas/templates/_form_macros.html

```diff
@@ -1,24 +1,30 @@
-{% macro with_errors(field) %}
-  {% if kwargs.pop('nolabel', '') %}
-    {% set label = False %}
-  {% else %}
-    {% set label = True %}
-  {% endif %}
+{% macro form_field(field) %}
+  {% set css_class = 'form-control ' + kwargs.pop('class', '') %}
   {% if field.errors %}
-    {% set css_class = 'form-control is-invalid ' + kwargs.pop('class', '') %}
-  {% else %}
-    {% set css_class = 'form-control ' + kwargs.pop('class', '') %}
+    {% set css_class = css_class + ' is-invalid ' %}
   {% endif %}
-
-  {% if label %}
+  {% if kwargs.pop('label', True) %}
     <label for="{{ field.name }}" class="mb-0 font-weight-bold">{{ field.label }}</label>
   {% endif %}
   {{ field(class=css_class, **kwargs) }}
-  
   {% if field.errors %}
-  <div class="invalid-feedback">
+    <div class="invalid-feedback">
       {{ field.errors|map('escape')|join(' ') }}
-  </div>
+    </div>
+  {% endif %}
+{% endmacro %}
+
+{% macro with_errors(field) %}
+  {% set css_class = 'form-control ' + kwargs.pop('class', '') %}
```

Comment:
If you leave that one here, then `class` will have been removed from the `kwargs` and it won't be available in the `form_field` macro. I suggest removing that line since we don't use `ccs_class` in that block anymore anyway.